File: src/management/server/SmartSignalsManagementApi/Responses/ListAlertrulesResponse.cs
//-----------------------------------------------------------------------
// <copyright file="ListAlertRulesResponse.cs" company="Microsoft Corporation">
//        Copyright (c) Microsoft Corporation.  All rights reserved.
// </copyright>
//-----------------------------------------------------------------------

namespace Microsoft.Azure.Monitoring.SmartAlerts.Appliance.ManagementApi.Responses
{ 
    using System.Collections.Generic;
    using Microsoft.Azure.Monitoring.SmartAlerts.Appliance.ManagementApi.Models;
    using Newtonsoft.Json;

    /// <summary>
    /// This class represents the GET alert rules operation.
    /// </summary>
    public class ListAlertRulesResponse
    {
        /// <summary>
        /// Gets or sets the alert rules
        /// </summary>
        [JsonProperty("alertRules")]
        public List<AlertRuleApiEntity> AlertRules { get; set; }
    }
}

File: src/management/client/src/api/models/ListAlertRulesResponse.ts
// -----------------------------------------------------------------------
// <copyright file="ListAlertRulesResponse.ts" company="Microsoft Corporation">
//        Copyright (c) Microsoft Corporation.  All rights reserved.
// </copyright>
// -----------------------------------------------------------------------

import AlertRule from '../../models/AlertRule';

/**
 * Represents the response model for the GET /resource endpoint
 */
export default class ListAlertRulesResponse {
    /**
     * The alert rules
     */
    public alertRules: ReadonlyArray<AlertRule>;
}